ci/utils.py

```diff
@@ -168,3 +173,84 @@ def upload_bundle(
     source = f"https://github.com/{repo_name}/releases/download/{release_tag}/{bundle_zip_filename}"
 
     return source
+
+
+def find_bundle_file(root_dir: str, file: str, suffix=("json", "yaml", "yml")):
+    # find bundle file with possible suffix
+    file_name = None
+    for name in suffix:
+        full_name = f"{file}.{name}"
+        if full_name in os.listdir(root_dir):
+            file_name = full_name
+
+    return file_name
+
+
+def produce_dataset(dataset_dir, dataset_size, input_shape):
+    for s in range(dataset_size):
+        test_image = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+        test_label = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+        image_filename = os.path.join(dataset_dir, f"image_{s}.nii.gz")
+        label_filename = os.path.join(dataset_dir, f"label_{s}.nii.gz")
+        nib.save(nib.Nifti1Image(test_image, np.eye(4)), image_filename)
+        nib.save(nib.Nifti1Image(test_label, np.eye(4)), label_filename)
```

Comment:
Maybe some bundles don't expect NIfTI file as input?
I would suggest to move the saving logic into every test case directly.

Thanks.

Reply:
Thanks, I reformatted the functions